manticore/platforms/linux.py

```diff
@@ -1,21 +1,22 @@
+import errno
 import fcntl
+import logging
+import os
+import random
+import struct
+
+from elftools.elf.elffile import ELFFile
 
-import cgcrandom
-import weakref
```

Comment:
looks like `weakref` was used in 1 place that is commented out. i don't see a point in keeping the commented code, so lets clean that too